graphite_api/render/glyph.py

```diff
@@ -1445,209 +1753,72 @@ def makeLabel(yValue):
-        seriesWithMissingValuesR = [
-            series for series in Rdata if None in series]
-
-        if self.params.get('drawNullAsZero') and seriesWithMissingValuesL:
-            yMinValueL = 0.0
-        else:
-            yMinValueL = safeMin([
-                safeMin(series) for series in Ldata
-                if not series.options.get('drawAsInfinite')])
-        if self.params.get('drawNullAsZero') and seriesWithMissingValuesR:
-            yMinValueR = 0.0
-        else:
-            yMinValueR = safeMin([
-                safeMin(series) for series in Rdata
-                if not series.options.get('drawAsInfinite')])
-
-        if self.areaMode == 'stacked':
-            yMaxValueL = safeSum([safeMax(series) for series in Ldata])
-            yMaxValueR = safeSum([safeMax(series) for series in Rdata])
-        else:
-            yMaxValueL = safeMax([safeMax(series) for series in Ldata])
-            yMaxValueR = safeMax([safeMax(series) for series in Rdata])
-
-        if yMinValueL is None:
-            yMinValueL = 0.0
-        if yMinValueR is None:
-            yMinValueR = 0.0
-
-        if yMaxValueL is None:
-            yMaxValueL = 1.0
-        if yMaxValueR is None:
-            yMaxValueR = 1.0
+        drawNullAsZero = self.params.get('drawNullAsZero')
+        stacked = (self.areaMode == 'stacked')
 
-        if 'yMaxLeft' in self.params:
-            yMaxValueL = self.params['yMaxLeft']
-        if 'yMaxRight' in self.params:
-            yMaxValueR = self.params['yMaxRight']
+        (yMinValueL, yMaxValueL) = dataLimits(self.dataLeft, drawNullAsZero,
+                                              stacked)
+        (yMinValueR, yMaxValueR) = dataLimits(self.dataRight, drawNullAsZero,
+                                              stacked)
 
-        if (
-            'yLimitLeft' in self.params and
-            self.params['yLimitLeft'] < yMaxValueL
-        ):
-            yMaxValueL = self.params['yLimitLeft']
-        if (
-            'yLimitRight' in self.params and
-            self.params['yLimitRight'] < yMaxValueR
-        ):
-            yMaxValueR = self.params['yLimitRight']
-
-        if 'yMinLeft' in self.params:
-            yMinValueL = self.params['yMinLeft']
-        if 'yMinRight' in self.params:
-            yMinValueR = self.params['yMinRight']
-
-        if yMaxValueL <= yMinValueL:
-            yMaxValueL = yMinValueL + 1
-        if yMaxValueR <= yMinValueR:
-            yMaxValueR = yMinValueR + 1
-
-        yVarianceL = yMaxValueL - yMinValueL
-        yVarianceR = yMaxValueR - yMinValueR
-        orderL = math.log10(yVarianceL)
-        orderR = math.log10(yVarianceR)
-        orderFactorL = 10 ** math.floor(orderL)
-        orderFactorR = 10 ** math.floor(orderR)
-        # we work with a scaled down yVariance for simplicity
-        vL = yVarianceL / orderFactorL
-        vR = yVarianceR / orderFactorR
+        # TODO: Allow separate bases for L & R Axes.
```

Comment:
Yes, it is.  It may disappear soon as development seems to have really picked up in the last few weeks upstream.